susemanager-utils/susemanager-sls/salt/supportdata/init.sls

```diff
@@ -0,0 +1,15 @@
+install-supportdata-command:
+  pkg.latest:
+{%- if grains['os_family'] == 'Suse' %}
+    - name: supportutils
+{%- else %}
+    - name: sos
+{%- endif %}
+
+
+gather-supportdata:
+  mgrcompat.module_run:
+    - name: supportdata.get
+    - cmd_args: pillar.get('arguments')
```

Comment:
Not sure here, do we need to wrap this with double brackets? don't think I've seen just raw usage of salt functions without any `{{ }}` (or outside of a `{ .. }` block, e.g. [0] or [1].

If you're sure that this works, or that my suggestion breaks something, feel free to ignore. 

```suggestion
    - cmd_args: "{{ pillar.get('arguments') }}"
```

[0] https://github.com/uyuni-project/uyuni/blob/master/susemanager-utils/susemanager-sls/salt/images/docker.sls#L14
[1] https://github.com/uyuni-project/uyuni/blob/master/susemanager-utils/susemanager-sls/salt/images/docker.sls#L12

Reply:
I just tried it and got this:
```
executing: ['/sbin/supportconfig', '-R', '/var/log/supportdata-20250414143507', "pillar.get('arguments')"]
```
Yes, we need this. Thanks